DuggaSys/analytic.js

```diff
@@ -1268,6 +1269,9 @@ function loadUserInformation(){
 				localStorage.setItem('analyticsLastUser', $(this).val());
 			} catch(err) { }
 
+			if(currentPage != "events" && currentPage != "fileEvents"){
```

Comment:
loginFail causes an infinite loop, the other pages seem to work fine after this change, I'd suggest adding this as well.